cdap-proto/src/main/java/co/cask/cdap/proto/MetricQueryResult.java

```diff
@@ -44,6 +46,8 @@ public long getEndTime() {
     return series;
   }
 
+  public int getResolution() { return resolution; }
```

Comment:
this should not pass checkstyle. there should be a newline before and after the ``return`` statement.

Btw, you may want to check if your IDE settings are up-to-date. You should get a checkstyle error in IDEA.